src/index.js

```diff
@@ -54,6 +54,11 @@ export default class RabbotClient {
     this.client = rabbot;
   }
 
+  // Function should recieve a queue message and return a gb-services style context.
+  setContextFunction(contextFunction) {
```

Comment:
This shouldn't just be a function - it should be callable from config, so probably a key on the config passed to the constructor